ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -408,13 +408,15 @@ gcloud "--project=${HOST_PROJECT}" dns record-sets transaction add \
   --name "_etcd-server-ssl._tcp.${CLUSTER_NAME}.${BASE_DOMAIN}." --ttl 60 --type SRV --zone "${PRIVATE_ZONE_NAME}"
 gcloud "--project=${HOST_PROJECT}" dns record-sets transaction execute --zone "${PRIVATE_ZONE_NAME}"
 
+MASTER_IGS="$(gcloud compute instance-groups list --filter network:${CLUSTER_NETWORK} --sort-by 'NAME' | sed 's/ /\n/g' | grep master)"
+read -r MASTER_IG_1 MASTER_IG_2 MASTER_IG_3 <<< "$(echo ${MASTER_IGS} | tr '\n' ' ')"
```

Comment:
The sort is fragile. It assumes (1) that the zones are unique and (2) that the zones are in lexigraphical order in the output of `gcloud compute regions describe`. The (1) is a fragile assumption that the previous code made as well. Nevertheless, I think it would be better to explicitly get each instance group individually for the specific zone.
```suggestion
MASTER_IG_0="$(gcloud compute instance-groups list --filter "network:${CLUSTER_NETWORK} AND name~^${INFRA_ID}-master-${ZONE_0}-" --format "value(name)")"
MASTER_IG_1="$(gcloud compute instance-groups list --filter "network:${CLUSTER_NETWORK} AND name~^${INFRA_ID}-master-${ZONE_1}-" --format "value(name)")"
MASTER_IG_2="$(gcloud compute instance-groups list --filter "network:${CLUSTER_NETWORK} AND name~^${INFRA_ID}-master-${ZONE_2}-" --format "value(name)")"
```
Note that we could probably get clever here with a for loop an indirect expansion of the variable names, but that is probably overkill for the sake of avoiding duplication for three lines of code.

Also, note that the names of the `MASTER_IG_X` variables are changed to be 0-indexed to match the indexing of the zone and instance names.  The `add-instances` command further down would need to be adjusted accordingly.